docs/index.rst

```diff
@@ -51,8 +51,15 @@ The ME-ICA pipeline originally performed both pre-processing and TE-dependent
 analysis of multi-echo fMRI data; however, ``tedana`` now assumes that you're
 working with data which has been previously preprocessed.
 
-.. _ME-ICA: https://github.com/me-ica/me-ica
+For a summary of multi-echo fMRI, which is the technology ``tedana`` builds on, 
+visit `Multi-echo fMRI`_. 
+
+For a detailed procedure of how ``tedana`` analyzes the data from multi-echo fMRI, 
+visit `Processing pipline details`_.
```

Comment:
```suggestion
visit `Processing pipeline details`_.
```

Reply:
@monicayao you can just commit the above suggestion instead of manually editing it.